pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -71,6 +71,17 @@ func checkPortAvailable(
 	return nil
 }
 
+func enableVlanFiltering(bridgeName string) {
+	args := strings.Split(fmt.Sprintf("link set dev %s type bridge vlan_filtering 1", bridgeName), " ")
+	log.Functionf("Calling command %s %v\n", "ip", args)
+	out, err := base.Exec(log, "ip", args...).CombinedOutput()
```

Comment:
Doesn't the netlink package have support for this?

Reply:
It has the support. I'll have to rebase to the latest code.

Reply:
Issues with pulling latest module code.

Reply:
Should we fix the issue with pulling latest eriknordmark/netlink and use netlink library for setting vlan info as a separate task?

Reply:
Honestly @gkodali-zededa -- I think the best approach for you is to do the rebase on the latest netlink as a separate PR -- that'll be much cleaner.

Reply:
yeah, will do that as separate PR.